xyz.audiostellar.AudioStellar.yml

```diff
@@ -0,0 +1,344 @@
+app-id: xyz.audiostellar.AudioStellar
+runtime: org.freedesktop.Platform
+runtime-version: '23.08'
+sdk: org.freedesktop.Sdk
+command: audiostellar
+finish-args:
+  # We need MIDI, Audio, USB, DRI, and realtime performances
+  - --device=all
+  
+  # We need X11, PulseAudio, JACK and Pipewire support
+  - --share=ipc
+  - --socket=x11
+  - --socket=pulseaudio
+  - --filesystem=xdg-run/pipewire-0  
+  
+  # We are using the OSC protocol
+  - --share=network
+  
+  - --filesystem=home
+  # Possible removable devices
+  - --filesystem=/mnt
```

Comment:
```suggestion
```

Reply:
isn't this a common mount point for some distros ?

Reply:
this is not for removable media.

Reply:
thanks!
done in the last commit